components/pkg-export-kubernetes/defaults/KubernetesManifest.hbs

```diff
@@ -1,15 +1,16 @@
+{{#if config}}
 ---
 ## Secret for initial configuration.
-#apiVersion: v1
-#kind: Secret
-#metadata:
-#  name: user-toml-secret
-#type: Opaque
-#data:
-#  # Each configuration item needs to be encoded in base64.
-#  # Plain text content of the secret: "port = 4444"
-#  user.toml: cG9ydCA9IDQ0NDQ=
-#---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: user-toml-secret
+type: Opaque
+data:
+  ## Each configuration item needs to be encoded in base64.
```

Comment:
Should we add a warning that this is just encoding and these files should not be shared if they contain sensitive configuration, or is it obvious?

Reply:
are people really expected to modify these generated files?

Reply:
but I would hope that if someone knows what base64 is and can encode things into that, they know that it's not encrypted.

Reply:
Most likely they will need to modify them to suit their needs. Yes, one would hope. :)